kythe/go/platform/vfs/vfs.go

```diff
@@ -63,6 +70,11 @@ type Writer interface {
 	// Create creates a new file for writing, as os.Create.
 	Create(ctx context.Context, path string) (io.WriteCloser, error)
 
+	// TempFile creates a new temp file returning a NamedWriteCloser to it. The
+	// name of the file is constructed from dir pattern and per
+	// ioutil.TempFile.
+	TempFile(ctx context.Context, dir, pattern string) (NamedWriteCloser, error)
```

Comment:
Should specify what dir and pattern mean.

Reply:
Done